overops_collector/app/collector.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
-set
 export TAKIPI_LISTEN_PORT=$PORT
 export TAKIPI_SERVER_NAME=$(echo $VCAP_APPLICATION | jq -r '.name')$INSTANCE_INDEX
 tar -xzvf takipi-latest.tar.gz
```

Comment:
This command isn't going to work, `takipi-latest.tar.gz` doesn't exist. The file was renamed `takipi-4.37.7.tar.gz` in this commit.